common/src/main/java/com/wynntils/wc/utils/scoreboard/objectives/ObjectiveManager.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.utils.scoreboard.objectives;
+
+import com.wynntils.core.WynntilsMod;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ObjectiveManager {
+    // §b is guild objective, §a is normal objective and §c is daily objective
+    public static final Pattern OBJECTIVE_PATTERN = Pattern.compile("^§([b|ac])[- ]\\s§7(.*): *§f(\\d+)§7/(\\d+)$");
```

Comment:
You don't need `|` as or in a `[]` group. (I think it is actually interpreted as the literal `|` character)